docs/application/dotnet/api/overview.md

```diff
@@ -56,4 +56,4 @@ The following table lists the supported Tizen platform-specific API namespaces.
 | Tizen.System       | Provides device-specific services, including status, system information and settings, control of haptic devices and sensor control and access. |
 | Tizen.Telephony    | Provides call, modem, network, and SIM information. |
 | Tizen.Uix          | Provides functions to recognize the speech and to synthesize voice from text and playing synthesized sound data. |
-| Tizen.Webview      | Provides functions to access web pages and web contents. |
+| Tizen.Webview      | Provides functions to access webpages and web contents. |
```

Comment:
Why did you change the words?

Reply:
@journey2w, according to our style guide we are directed to use "webpages" as one word.

Reply:
There are lots of "a web page" and "web pages" in our guides. I didn't see your style guide as well as it is not standard that you change just one page.